doc/nrf/ug_thingy91.rst

```diff
@@ -170,9 +170,9 @@ Building and programming using |VSC|
 
 Complete the following steps after installing the |nRFVSC|:
 
-.. |sample_path_vsc| replace:: :file:`samples/nrf9160/cloud_client`
+.. |sample_path_vsc| replace:: :file:`samples/nrf9160/aws_iot`
 
-.. |vsc_sample_board_target_line| replace:: you must use the build target ``thingy91_nrf9160`` or ``thingy91_nrf9160_ns`` when building the application code for the nRF9160 SiP and the build target ``thingy91_nrf52840`` when building the application code for the onboard nRF52840 SoC.
+.. |vsc_sample_board_target_line| replace:: you must use the build target ``thingy91_nrf9160`` or ``thingy91_nrf9160_ns`` when building the application code for the nRF9160 SiP and the build target ``thingy91_nrf52840`` when building the application code for the onboard nRF52840 SoC
```

Comment:
`thingy91_nrf9160` and `thingy91_nrf52840` is not relevant in this context. The `thingy91_nrf52840` should be documented in a separate guide on how to flash the connectivity_application application.

Reply:
In this userguide we explain about nRF52840 SoC also. So, I think we should have this details also.

http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91.html#building-and-programming-from-the-source-code